src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,10 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
```

Comment:
It's a good practice to define default props for your functional components. This will prevent your app from crashing in case 'comments' prop is undefined.